spec.yaml

```diff
@@ -140,15 +137,54 @@ paths:
               description: >
                 Indication for the client when the service could be requested
                 again in HTTP Date format as used by the 
-
                 Internet Message Format [RFC5322] (e.g. `Wed, 21 Oct 2015
                 07:28:00 GMT`) or the number of seconds 
-
                 (e.g. 3000 if you the service is expected to be available again
                 within 50 minutes).
               type: string
           schema:
             $ref: '#/definitions/Error'
+  /invites/forward:
```

Comment:
For the record, there was also the idea to rename `/forward` as well, but I'm not sure something like `/deliver` would be much better.

On the other hand, I support renaming `/accept` to `/accepted` as this is really a message sent back to inform that the invitation was accepted.

Reply:
Discussing with @michielbdejong, the `forward` action is actually called `accept` in the OC/NC apps, and there's no explicit OCM `accept` action in those apps.